fiftyone/core/session.py

```diff
@@ -27,6 +27,7 @@
 import fiftyone.utils.templates as fout
 from fiftyone.core.state import StateDescription
 
+import pkg_resources
```

Comment:
Standard library imports go in the first block of imports. After `import logging` in this case.

Reply:
Thanks for the comments, I applied them